modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/DataEngineFormBuilder.js

```diff
@@ -65,6 +87,32 @@ const DataEngineFormBuilder = () => {
 								},
 							]}
 						/>
+						<MultiPanelSidebar
+							createPlugin={({
+								panel,
+								sidebarOpen,
+								sidebarPanelId,
+							}) => ({
+								dispatch,
+								panel,
+								sidebarOpen,
+								sidebarPanelId,
+							})}
+							currentPanelId={sidebarPanelId}
+							onChange={({sidebarOpen, sidebarPanelId}) =>
+								dispatch({
```

Comment:
Instead of leaving this dispatch here, just creating a state on the DataEngineFormBuilder since the Sidebar will just be used on the DataEngine :)

Reply:
Done!